core/server/utils/version.js

```diff
@@ -0,0 +1,8 @@
+var packageInfo = require('../../../package.json'),
```

Comment:
suggestion: 
i find the name `version` a bit too general. 
Maybe call it `ghost-version.js`?

Example: I require the file `version  = require('./utils/version')`. When i am reading `verison.full` in the middle of some code, this is hard to know what `version.full` means.